redisgraph/crates/wasccgraph-actor/src/conversions.rs

```diff
@@ -0,0 +1,122 @@
+use crate::client_type_error;
```

Comment:
Again, this stuff belongs in the guest feature of the `actor_graphdb` crate, as none of this is Redis-specific...this abstraction that allows for the tuple-based query of a resultset should be available to all actors using graphdb, regardless of whether the implementation is Neo4j or redisgraph, etc.